crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -213,23 +218,31 @@ async fn symbolicate_js_frame(
                 None => &Err(CacheError::NotFound),
             };
 
-            apply_source_context_from_artifact(&mut frame, source_file);
+            apply_source_context_from_artifact(&mut frame, source_file).ok();
         }
     }
 
     Ok(frame)
 }
 
-fn apply_source_context_from_artifact(frame: &mut JsFrame, file: &CacheEntry<CachedFile>) {
+fn apply_source_context_from_artifact(
+    frame: &mut JsFrame,
+    file: &CacheEntry<CachedFile>,
+) -> Result<(), JsModuleErrorKind> {
     if let Ok(file) = file {
         apply_source_context(frame, &file.contents)
     } else {
-        // TODO(sourcemap): report missing source?
+        Err(JsModuleErrorKind::MissingSourcemap)
```

Comment:
```suggestion
        Err(JsModuleErrorKind::MissingSource)
```

We are either missing the minified or the original source file here, as we are not calling this function with a *sourcemap*.